README.md

```diff
@@ -69,3 +69,33 @@ and `nix-env` to install the application:
 ```
 $ nix-env --file default.nix --install
 ```
+
+### Cabal
+
+Install development packages for system library dependencies
+first. These are needed to compile the Haskell notmuch
+bindings. Package names may vary between different distributions:
+
+* libtalloc-devel
+* notmuch-devel
+
+Make sure you have at least one of our supported GHC and Cabal-install >= 2.2
+installed. From a cloned checkout you then go:
+
+```
+# Updates the package list needed for cabal to download dependencies
+$ cabal update
```

Comment:
``should be new-update``, to go with ``new-build``

Reply:
See.. I thought that was one of the commands which didn't change. Will update.